### PR TITLE
import.py fix

### PR DESCRIPTION
**Pull request by natashamaniar**

### CODE REVIEW

File: import.py [hunk 1]
```diff
@@ -3,14 +3,36 @@
...
 def import_(project_id, filename): 
     
     client = dialogflow.AgentsClient()
     parent = client.project_path(project_id)
     with open(filename, 'rb') as file_data:
         bytes_content = file_data.read() 
+
     client.restore_agent(parent=parent,agent_content=bytes_content) 
 if __name__ == "__main__": 
+    repo = Repo('/Users/natasha/serenity-intentmatch')
```
Comment (jkaardal) [inline import.py:17]: Can't hardcode this path. Instead use `os.path.abspath(os.path.dirname(__file__))`.

File: import.py [hunk 2]
```diff
@@ -3,14 +3,36 @@
...
+        '--project-id',
+        help='Project/agent id.  Required.',
+        required=True) 
+    parser.add_argument(
+        '--tag-name',
+        help='Tag name-version.  Required.',
+        required=True) 
+    args = parser.parse_args()
+
+    with open('serenity-agent.zip', 'wb') as archive_file:
```
Comment (jkaardal) [inline import.py:34]: Should check if this file exists and if so raise an exception.

File: import.py [hunk 3]
```diff
@@ -3,14 +3,36 @@
...
+    args = parser.parse_args()
+
+    with open('serenity-agent.zip', 'wb') as archive_file: 
+        repo.archive(archive_file, format='zip',treeish=args.tag_name,path='serenity-DF-agent') 
     # git checkout tag-name 
-    shutil.make_archive('serenity-agent', 'zip', 'serenity-DF-agent')
-    import_('serenity-test-sancao','serenity-agent.zip')    
\ No newline at end of file
+    #shutil.make_archive('serenity-agent', 'zip', 'serenity-DF-agent') 
+    import_(args.project_id,'serenity-agent.zip')
```
Comment (jkaardal) [inline import.py:38]: Delete the zip file with `os.remove()` when finished uploading.

File: import.py (hunk 2 above)
Comment (natashamaniar) [reply import.py:35]: when i open the file in write mode it opens a new file so I wouldn't have to check if it already exists

**Pull request merged**